NEWS.md

```diff
@@ -11,6 +11,14 @@
 
 * Failure locations are now formated as R error locations.
 
+* The [Catch](https://github.com/philsquared/Catch) unit
+  testing library can now be used for unit testing of C++
+  files. You can place C++ files within the `tests/testthat/cpp`
```

Comment:
It looks like `find_test_scripts` does the right thing: `files <- dir(path, "^test.*\\.[rR]$", full.names = TRUE)`. I'll follow its cue.